routes/web.php

```diff
@@ -35,3 +32,10 @@
 Route::post('/post', 'EarhquakesController@post')->name('post');
 Route::get('/users', 'UserController@users');
 Route::get('/mail/mailable/send', 'MailController@MailNotification');
+Route::post('/post', 'Earhquakescontroller@post')->name('post');
+Route::get('/contribution', 'ContributionController@index');
+Route::post('/contribution/add', 'ContributionController@add');
+Route::post('/contribution/delete/{id}', 'ContributionController@delete');
```

Comment:
The convention for Route is like this:
`contribution/{id}/delete`

The `id` is in before delete because it indicates that we want to perform an action (which is delete) to the contribution with an `id` of something.